src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/main/SimulationMenu.java

```diff
@@ -0,0 +1,101 @@
+package edu.kit.pse.osip.simulation.view.main;
+
+import edu.kit.pse.osip.core.utils.language.Translator;
+import edu.kit.pse.osip.simulation.controller.AbstractMenuSettingsButton;
+import edu.kit.pse.osip.simulation.controller.AbstractMenuAboutButton;
+import edu.kit.pse.osip.simulation.controller.AbstractMenuControlButton;
+import edu.kit.pse.osip.simulation.controller.AbstractMenuHelpButton;
+import javafx.scene.control.Button;
+import javafx.scene.control.Menu;
+import javafx.scene.control.MenuBar;
+import javafx.scene.control.MenuItem;
+import javafx.scene.layout.GridPane;
+
+/**
+ * This class is the MenuBar at the top of the simulation view. It provides options to control the
+ * simulation, change the settings and get help.
+ *
+ * @author Niko Wilhelm
+ * @version 1.0
+ */
+public class SimulationMenu extends GridPane {
+
+    /**
+     * Menu containing all buttons leading to ways to conrol the simulation.
+     */
+    private Button buttonControl;
+
+    /**
+     * Menu containing all buttons leading to ways to change the settings of the simulation.
+     */
+    private Button buttonSettings;
+
+    /**
+     * Menu containing all buttons for help.
+     */
+    private Menu menuOther;
+
+    /**
+     * Button in the help menu for showing the help dialog.
+     */
+    private MenuItem menuButtonHelp;
+
+    /**
+     * Button in the about menu for showing the about dialog.
+     */
+    private MenuItem menuButtonAbout;
+
+    /**
+     * Creates and initializes the menu for the simulation view.
+     */
+    protected SimulationMenu() {
+        Translator trans = Translator.getInstance();
+        buttonControl = new Button(trans.getString("simulation.view.menu.control"));
+
+        buttonSettings = new Button(trans.getString("simulation.view.menu.settings"));
```

Comment:
Hmm das stimmt... Aber in einer Toolbar und mit Icon :D

...was jetzt nicht heißen soll, dass du Icons malen sollst